src/data/markdown/docs/02 javascript api/07 k6-experimental/02 redis/10 Client/Client-getdel.md

```diff
@@ -35,11 +35,10 @@ const redisClient = new redis.Client({
   password: redis_password,
 });
 
-export default function () {
-  redisClient
-    .set('mykey', 'oldvalue', 0)
-    .then((_) => redisClient.getSet('mykey', 'newvalue'))
-    .then((_) => redisClient.getDel('mykey'));
+export default async function () {
+    await redisClient.set('mykey', 'oldvalue', 0);
+    await redisClient.getSet('mykey', 'newvalue');
+    await redisClient.getDel('mykey');
```

Comment:
nit: we do check in most other tests that what we do has the expected result. But here we just call stuff.

Given that this are not unit/integration tests of the redis API it is actually a mute point, but still, seems off.

Reply:
you're totally right, gonna add some checks 👍🏻